src/views/open-api-docs-view/components/property-details/index.tsx

```diff
@@ -47,18 +48,29 @@ export function PropertyDetails({
 					<span className={s.baseType}>{type}</span>
 					{isRequired ? <Badge text="Required" color="highlight" /> : null}
 				</div>
-				{description ? (
-					<div
-						className={s.baseDescription}
-						dangerouslySetInnerHTML={{ __html: description }}
-					/>
-				) : null}
 			</div>
-			{nestedProperties?.length > 0 ? (
-				<ListNestedProperties
-					nestedProperties={nestedProperties}
-					depth={depth}
-				/>
+			{description || hasNestedProperties ? (
+				<div>
```

Comment:
Yeah exactly it's for layout! Using flex layouts here to avoid margin logic and conditional margin styles, but the extra divs definitely feel like a bit of a downside 😅